aspnetcore/web-api/index.md

```diff
@@ -392,6 +392,28 @@ The automatic creation of a `ProblemDetails` instance is disabled when the <xref
 
 ::: moniker-end
 
+## Define supported request content types with the [Consumes] attribute
+
+By default, an action supports all available request content types. For example, if an app is configured to support both JSON and XML [input formatters](xref:mvc/models/model-binding#input-formatters), an action supports multiple content types, including `application/json` and `application/xml`.
```

Comment:
Is by default correct? I didn't understand @serpent5  version which is  "Without further configuration,"

Reply:
Yeah, sorry, my wording was clunky. By default seems better to me.

Reply:
@serpent5  really nice work on your part. This seems to be very useful.